packages/patternfly-4/react-core/README.md

```diff
@@ -1,45 +1,32 @@
 # @patternfly/react-core
 
-This package provides Core PatternFly components for the [PatternFly 4][patternfly-4].
+![npm badge](https://img.shields.io/npm/v/@patternfly/react-core.svg?style=for-the-badge)
 
-### Prerequisite
+This package provides functionality and styles for essential PatternFly components. It depends on [PatternFly 4 CSS][patternfly-4]. The repo can be found [here][patternfly-repo].
 
-#### Node Environment
 
-This project currently supports Node [Active LTS](https://github.com/nodejs/Release#release-schedule) releases. Please stay current with Node Active LTS when developing patternfly-react.
+## Installing
 
-For example, to develop with Node 8, use the following:
+Use your favorite package manager to install our package from npm: 
 
-```
-nvm install 8
-nvm use 8
-```
-
-This project also requires a Yarn version of >=1.6.0. The latest version can be installed [here](https://yarnpkg.com/).
-
-### Installing
-
-```
-yarn add @patternfly/react-core
-```
-
-or
+`yarn add @patternfly/react-core`
 
-```
-npm install @patternfly/react-core --save
-```
+`npm install @patternfly/react-core --save`
```

Comment:
I think we need an or so someone does not do both.

Reply:
We should add back the `pr`.